zerver/openapi/zulip.yaml

```diff
@@ -5765,6 +5765,79 @@ paths:
                                   "op": "remove",
                                   "saved_snippet_id": 17,
                                 }
+                            - type: object
+                              additionalProperties: false
+                              description: |
+                                Event sent to a user's clients when reminders
+                                are created.
+
+                                **Changes**: New in Zulip 11.0 (feature level ZF-f1fd6c).
+                              properties:
+                                id:
+                                  $ref: "#/components/schemas/EventIdSchema"
+                                type:
+                                  allOf:
+                                    - $ref: "#/components/schemas/EventTypeSchema"
+                                    - enum:
+                                        - reminders
+                                op:
+                                  type: string
+                                  enum:
+                                    - add
+                                reminders:
+                                  type: array
+                                  description: |
+                                    An array of objects containing details of the newly created
+                                    reminders.
+                                  items:
+                                    $ref: "#/components/schemas/Reminder"
+                              example:
+                                {
+                                  "type": "reminders",
+                                  "op": "add",
+                                  "reminders":
+                                    [
+                                      {
+                                        "scheduled_message_id": 17,
```

Comment:
I kinda feel like this should be `reminder_id`, not `scheduled_message_id`, here. 

Have we discussed this format in the "api design" channel? Probably worth a quick post there to a "reminder events" topic with this format there for confirmation.

Reply:
Started discussion [here](https://chat.zulip.org/#narrow/channel/378-api-design/topic/Delete.2FEdit.20a.20reminder/with/2207011)